src/database.py

```diff
@@ -0,0 +1,10 @@
+import psycopg2
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+try:
+    conn = psycopg2.connect(f"dbname='postgres' user='{os.getenv('POSTGRES_USER')}' host='database' password='{os.getenv('POSTGRES_PASSWORD')}'")
```

Comment:
Can you use `POSTGRES_DB` env variable instead of hardcoded 'postgres' name?

Maybe also create POSTGRES_HOST env variable and reference it here?